documentation/document.css

```diff
@@ -182,4 +175,8 @@
     .footer-center i {
         font-size: 5rem;
         padding-left: 2rem;
+    }
+    
+    .icon {
+        color: black
     }
```

Comment:
Give an extra empty line after line 182, to prevent any issues that may crop up later
Refer to here: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

Reply:
Yes, will do it in next PR